source/particle/property/interface.cc

```diff
@@ -30,6 +30,80 @@ namespace aspect
   {
     namespace Property
     {
+      ParticlePropertyInformation::ParticlePropertyInformation()
+      {}
```

Comment:
If you write it like this, then you don't actually need the constructor. But I think you probably want to explicitly initialize the scalar numbers here.